source/common/protobuf/utility.cc

```diff
@@ -31,7 +31,8 @@ void MessageUtil::loadFromFile(const std::string& path, Protobuf::Message& messa
 }
 
 Json::ObjectSharedPtr WktUtil::getJsonObjectFromStruct(const Protobuf::Struct& message) {
-  Protobuf::util::JsonOptions json_options;
+  Protobuf::util::JsonPrintOptions json_options;
+  json_options.preserve_proto_field_names = true;
```

Comment:
Could leave a comment that this done to make debugging easier when looking at the JSON (since this is only being printed for the purpose of protobuf parsing).